fili-core/src/main/java/com/yahoo/bard/webservice/web/OutstandingRateLimitedRateLimitRequestToken.java

```diff
@@ -0,0 +1,145 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Meter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.security.Principal;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * RateLimitRequestToken for successfully bound request.
+ */
+public class OutstandingRateLimitedRateLimitRequestToken extends RateLimitRequestToken {
```

Comment:
I see two `RateLimit` in name. Are the different?

Reply:
fixed